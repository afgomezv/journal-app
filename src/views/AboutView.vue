<template>
  <div class="about">
    <h1>This is an about page</h1>
  </div>
</template>

//describe( pruebas en el About View)
// test(debe de renderizar el componente correctamente)

