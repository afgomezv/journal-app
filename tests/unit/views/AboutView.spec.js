import { shallowMount } from "@vue/test-utils";
import AboutView from "@/views/AboutView";

describe("AboutView Component", () => {
  test("debe de hacer match con el snapshot", () => {
    const wrapper = shallowMount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
