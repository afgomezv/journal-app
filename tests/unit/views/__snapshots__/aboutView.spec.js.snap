// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView Component debe de hacer match con el snapshot 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
